docs/participate/03_treasury_proposal.md

```diff
@@ -11,6 +11,46 @@ The first step in applying for a Treasury grant is either to join the community
 This will help you get valuable community feedback throughout the process.
 It also gives Council members an open and transparent way to measure community sentiment.
 
+When submitting for discussion, it is best practice to include a proposal document.
+[A reference document](https://docs.google.com/document/d/1nz5GNjTL5inaI5s9djIki7DUp5zyPJV9dEI9K06kJ58/edit) can be found in the KILT Protocol discussions this can be used as a template for future proposals.
```

Comment:
I think we have better documents we can refer to, don't we? Like the Dwellir proposals? Or the Galani one?

Reply:
I have updated with the galani one.